datasets/librispeech.py

```diff
@@ -169,7 +169,7 @@ def run(self):
                 continue
             speaker = corpus.Speaker()
             speaker.name = speaker_id
-            speaker.attribs["gender"] = speaker_info[0].lower()
+            speaker.attribs["gender"] = ["male" if speaker_info[0] == "M" else "female"]
```

Comment:
I think this should be a `str` not a `list[str]`, right?

Reply:
Oh... yes sure